engine/schema/src/com/cloud/vm/dao/VMInstanceDao.java

```diff
@@ -151,8 +151,5 @@
 
     boolean isPowerStateUpToDate(long instanceId);
 
-    /**
-     * List running VMs which host and lastHost id are the same
-     */
-    List<VMInstanceVO> listRunningVmsByHostAndLastHostSameId(long hostId);
+    List<VMInstanceVO> listNonMigratingVmsByHostEqualsLastHost(long hostId);
```

Comment:
hm, that is not very clear, the name is stating cause and conclusion. I suggest listVmsByHostEqualsLastHost()